### PR TITLE
Azure createUiDefinition plumbing and first pass

### PR DESCRIPTION
**Pull request by silasbw**
### Description

createUiDefinition.json and bundle building for managed Azure applications.

### Tested

Tested manually with [createUiDefinition sandbox](https://portal.azure.com/?feature.customPortal=false#blade/Microsoft_Azure_CreateUIDef/SandboxBlade) and by creating a service catalog entry.

### CODE REVIEW

File: packages/azure/createUiDefinition.json [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+         "subLabel": {
+           "preValidation": "Configure cluster",
+           "postValidation": "Done"
+         },
+         "bladeTitle": "Cluster settings",
+         "elements": [
+           {
+             "name": "userObjectId",
+             "type": "Microsoft.Common.TextBox",
+             "label": "userObjectId",
```
Comment (codyborn) [inline packages/azure/createUiDefinition.json:22]: Could you provide a helpful hint for the users to understand better what these fields are used for?

File: packages/azure/createUiDefinition.json [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+           },
+           {
+             "name": "validatorAccountAddress",
+             "type": "Microsoft.Common.TextBox",
+             "label": "validatorAccountAddress",
+             "visible": true
+           },
+           {
+             "name": "validatorPrivateKey",
+             "type": "Microsoft.Common.TextBox",
```
Comment (codyborn) [inline packages/azure/createUiDefinition.json]: I think you can make this a password input to hide the key

File: packages/azure/createUiDefinition.json [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+         "elements": [
+           {
+             "name": "attesterPostgreSQLUsername",
+             "type": "Microsoft.Common.TextBox",
+             "label": "attesterPostgreSQLUsername",
+             "visible": true
+           },
+           {
+             "name": "attesterPostgreSQLPassword",
+             "type": "Microsoft.Common.TextBox",
```
Comment (codyborn) [inline packages/azure/createUiDefinition.json]: Password field as well

**Review by codyborn — APPROVED** [review]

File: packages/azure/createUiDefinition.json (hunk 1 above)
Comment (silasbw) [reply packages/azure/createUiDefinition.json:22]: oh, I'll remove this completely. I originally thought I needed to add an access policy for the user to access the Key Vaults at creation time (*e.g.*, if the ever wanted to read a secret value).  The user can create an access policy at any time that allows them to access secrets, so this isn't actually needed.

Comment (codyborn) [reply packages/azure/createUiDefinition.json:22]: Oh sorry, I meant that we can add a "tooltip" for all of these fields to give a human-readable description. There's a separate property for this:
`"toolTip": "The object Id of the AAD identity that will have access to Key Vault"`

Comment (silasbw) [reply packages/azure/createUiDefinition.json:22]: Ah, OK -- yes, I'll add tooltips for everything.

Related: https://github.com/celo-org/celo-monorepo/pull/3022

**Pull request merged**